src/lib/y2users/linux/writer.rb

```diff
@@ -225,6 +228,21 @@ def add_group(group, issues)
         log.error("Error creating group '#{group.name}' - #{e.message}")
       end
 
+      # Applies changes for the login settings
+      def edit_login(_issues)
+        return unless config.login?
+
+        # resetting Autologin settings
+        Yast::Autologin.Disable
```

Comment:
so if it is not configured, it is not disabled?

Reply:
Exactly, if the login is unknown, then the writer should do nothing.